### ISSUE TITLE
Inconsistent lemmatization?

### ISSUE THREAD

**Issue by leoalenc** [LFG-PTBR/MorphoBr#79]
À forma _fornalha_ são atribuídos dois lemas:

```
nouns/nouns-ab.dict:fornalha	fornalha+N+F+SG
nouns/nouns-ab.dict:fornalha	forno+N+AUG+F+SG
```

No segundo caso, a forma é analisada como aumentativo de _forno_. Justifica-se essa dupla análise? No Houaiss, temos _fornalha_ e _forno_ como entradas separadas, uma vez que o significado da primeira palavra não se resume à noção de aumentativo da segunda. Parece que os falantes não têm mais consciência de que _-alh-_ é um sufixo aumentativo.
As seguintes formas recebem apenas um lema cada, embora contenham sufixos diminutivos:

```
nouns/nouns-aa.dict:cabrito	cabrito+N+M+SG
nouns/nouns-ac.dict:pernoca	pernoca+N+F+SG
```
 
Aparentemente, há uma incongruência com o exemplo anterior. Por outro lado, temos:

```
nouns/nouns-aa.dict:cabrita	cabra+N+DIM+F+SG
nouns/nouns-aa.dict:cabrita	cabrita+N+F+SG
nouns/nouns-aa.dict:cabrita	cabrita+N+M+SG
nouns/nouns-aa.dict:cabrita	cabrito+N+F+SG
```

**Comment by vcvpaiva** [comment]
>Parece que os falantes não têm mais consciência de que -alh- é um sufixo aumentativo.

Concordo! mas acho q seria melhor manter o 'diminutivo' de cabrito, cabrita e pernoca.

**Comment by arademaker** [comment]
@leoalenc qual a conclusão aqui?

**Comment by vcvpaiva** [comment]
@arademaker  acho um erro tirar a entrada de cabrita como diminutivo de cabra. o sentido existe e e' mais padrao do q o informal https://www.dicionarioinformal.com.br/cabrita/

**Comment by arademaker** [comment]
Relacionado a discussão iniciada em #95 sobre escopo do recurso. Em acho que considerar formas coloquiais pode ser bem complicado. De fato, https://dicionario.priberam.org/cabrita identifica como DIM de cabra... @leoalenc ?

**Comment by leoalenc** [comment]
> Relacionado a discussão iniciada em #95 sobre escopo do recurso. Em acho que considerar formas coloquiais pode ser bem complicado. De fato, https://dicionario.priberam.org/cabrita identifica como DIM de cabra... @leoalenc ?

@arademaker, na minha versão do MorphoBr, anterior às suas últimas alterações, tenho:

`grep -P "cabritas?\t" nouns-a*`

> nouns-aa.dict:cabrita	cabra+N+DIM+F+SG
> nouns-aa.dict:cabrita	cabrita+N+F+SG
> nouns-aa.dict:cabrita	cabrita+N+M+SG
> nouns-aa.dict:cabrita	cabrito+N+F+SG
> nouns-aa.dict:cabritas	cabra+N+DIM+F+PL
> nouns-aa.dict:cabritas	cabrita+N+F+PL
> nouns-aa.dict:cabritas	cabrita+N+M+PL
> nouns-aa.dict:cabritas	cabrito+N+F+PL
  
Deveriam ser eliminadas:

> nouns-aa.dict:cabrita	cabrita+N+M+SG
> nouns-aa.dict:cabritas	cabrito+N+F+PL

Ficam:

Diminutivo de _cabra_, ou seja, "cabra pequena":

> nouns-aa.dict:cabrita	cabra+N+DIM+F+SG
> nouns-aa.dict:cabritas	cabra+N+DIM+F+PL

Formas do lema _cabrita_, com sentido diferente, não diminutivo:
> nouns-aa.dict:cabrita	cabrita+N+F+SG
> nouns-aa.dict:cabritas	cabrita+N+F+PL

https://www.infopedia.pt/dicionarios/lingua-portuguesa/cabrito
https://www.infopedia.pt/dicionarios/lingua-portuguesa/cabrita
